### PR TITLE
Project Setup

### PR DESCRIPTION
**Pull request by henask12**
**Pull Request Summary:**

## General Requirements

- [x] Code is free of linter errors.
- [x] Followed Gitflow workflow.

## Project Requirements

- [x] Created a new app with a meaningful name.
- [x] Initialized Git for version control.
- [x] Set up PostgreSQL database.
- [x] Designed and executed schema.

    - [x] Followed ERD for naming.
    - [x] Added foreign keys where necessary.
    - [x] Created indexes for foreign keys.

### CODE REVIEW

**Review by Whoistolu — COMMENTED** [review]
Hi @henask12 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### You have submitted a project, that does not have the Microverse recommended professional README. Kindly add this as it is part of the project requirements.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,24 +1,161 @@
...
+
+I would like to express my heartfelt gratitude to [Microverse](https://github.com/microverseinc) for providing us with the opportunityto work on this project. and I would like to extend my appreciation to my fellow Coding Partner who have played a significant role in making this project a successful.
+
+<p align='right'>(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a id="license"></a>
+
+This project is [MIT](./MIT) licensed.
```
Comment (abdulhamiid) [inline README.md]: Please fix the broken license link for a professional repository 😊

File: README.md [hunk 2]
```diff
@@ -1,24 +1,161 @@
...
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a id="authors"></a>
+
+👤 **Henok Mekonnen**
+
+- GitHub: [@githubhandle](https://github.com/henask12)
+- Twitter: [@twitterhandle](https://twitter.com/henok_mekonnen3)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/henokmekonnen1)
```
Comment (abdulhamiid) [inline README.md]: image

For a professional repository, let's avoid broken links in our readme, kindly fix the broken link in the `Authors` section 👍

File: db/schema.rb [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  end
+
+  create_table "users", force: :cascade do |t|
+    t.string "name"
+    t.string "photo"
+    t.text "bio"
+    t.integer "posts_counter"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
```
Comment (abdulhamiid) [inline db/schema.rb]: The implementation and method name for the provided sample codes should work. 👀, but here, I am getting errors.

image

`rails c` is the intended way to open a console for your app. It loads rails itself, all your `models`, and everything else needed.

I would suggest you create `user.rb` in `app/models/user.rb`

```ruby
class User < ApplicationRecord
end
```

As well as the remaining models, `post.rb`, `comment.rb` and `like.rb` for the app to work🤓🛠

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @henask12 👋, 
Alt Text
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Descriptions are descriptive
- ✔️ Install Ruby on Rails framework.
- ✔️ Use Ruby Gems as a software package system.

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers, and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @henask12, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [x] All migrations setup correctly :white_check_mark: 
- [x] Application functions as expected :white_check_mark: 
- [x] Repository has a professional readme file :white_check_mark: 
- [x] Pull request has a descriptive title and summary :white_check_mark: 
- [x] Implemented all the changes from the previous reviewer :white_check_mark:     

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**